src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>todo list</title>
+    <link rel="stylesheet" href="../src/style.css">
```

Comment:
- [ ] Well done :clap: you did a very nice job :+1: but there are some problems in your project I think you are not starting your project by using the `npm start` command if you check your project after running this command your CSS is not applied on your project this is because of you didn't import your `CSS file` in `index.js` file check below :point_down:  my video for more clarification :slightly_smiling_face: 

[screen-capture (17).webm](https://github.com/Rifat8080/To-do-list/assets/45226848/5b118d50-7fd2-49e7-a781-70712ae1ada8)